R/calculate.scaling.R

```diff
@@ -1,13 +1,12 @@
 calculate.scaling <- function(data.matrices) {
```

Comment:
**suggestion : (non-blocking)**

Consider resolving the warnings in this script.

Reply:
Does this mean we need to transform any input to the acceptable form?

Reply:
@maotian06 can you see the warnings from CICD here? https://github.com/uclahs-cds/package-iSubGen/pull/5/files/c674c07dce508aef2523dd09b4a9897b6ef48c9c#diff-320b64c68ade3bb58175a80bd01c3fe266a6035ea970651132a916eaaf1bbe4a

Reply:
@maotian06 can you see the warnings from CICD here? https://github.com/uclahs-cds/package-iSubGen/pull/5/files/c674c07dce508aef2523dd09b4a9897b6ef48c9c#diff-320b64c68ade3bb58175a80bd01c3fe266a6035ea970651132a916eaaf1bbe4a

Reply:
@tyamaguchi-ucla I have fixed the CICD warning! I was working on that while you commented.  Thanks for the reminder~